.gitignore

```diff
@@ -0,0 +1,23 @@
+# LaTeX temporary files
+*.aux
+*.log
+*.toc
+
+# PDF output - usually a bad idea to keep this in Git
+*.pdf
```

Comment:
I usually gitignore only the output pdf -- figure pdfs will get ignored with this

Reply:
figure pdfs will inside each file '00_xx'.. will this comment still ignore them? if that's the case, we gotta change them -- cuz I was not sure

Reply:
yep, it will, this ignores all pdfs

Reply:
fuck, how can we change it so that it ignores the only 'main' pdf then

Reply:
replace *.pdf with main.pdf

Reply:
lol